### PR TITLE
[AdvancedSearch] Add a tip about how to change the AdvancedSearch component in function of the selected scope

### PR DESCRIPTION
**Pull request by hnomichi**
## Description

Add an example about how to connect a parent component to the search store and update an advanced search props given the selected scope.

### CODE REVIEW

**Comment by pierr** [comment]
@hnomichi Can you use english inside your PR.
Also use the Pull request template, it would be great!

Thanks for your contribution.

File: src/page/search/advanced-search/README.md [hunk 1]
```diff
@@ -168,6 +168,62 @@ Evénement à définir par l'utilisateur.
...
+import connect from 'focus-components/behaviours/store/connect';
+
+// web components
+import {component as AdvancedSearch} from 'focus-components/page/search/advanced-search';
+import {advancedSearchStore} from 'focus-core/search/built-in-store';
+
+//Autres propriétés passées au composant, qui ne changent pas en fonction du scope
+import {configuration} from './configuration';
+
+class AdvancedSearchView extends Component {
```
Comment (pierr) [inline src/page/search/advanced-search/README.md]: It could be a pure function

`const AdvancedSearchView=  props => {...}`

Comment (pierr) [inline src/page/search/advanced-search/README.md]: You just have a render function

**Comment by pierr** [comment]
Also you fork seems to be unsynchronized from focus.

**Comment by hnomichi** [comment]
Updated.
I have changed the component declaration into a stateless function, I have updated the PR title in english, and I have updated my branch.

**Comment by pierr** [comment]
@hnomichi Thanks. 
It is better to rebase the branch instead of using the github merge button (it adds merge commits)
Can you resync your branch again.

**Comment by pierr** [comment]
@Ephrame do you agree with the doc content

File: src/page/search/advanced-search/README.md [hunk 2]
```diff
@@ -168,6 +168,57 @@ Evénement à définir par l'utilisateur.
...
+	)
+}
+AdvancedSearchView.displayName = 'AdvancedSearch';
+AdvancedSearchView.propTypes = {
+	scope: PropTypes.string.isRequired
+	};
+
+const connector = connect(
+    [{store: advancedSearchStore, properties: ['scope']}],
+    (props) => {return advancedSearchStore.getValue()}
```
Comment (pierr) [inline src/page/search/advanced-search/README.md]: uesless `(props) => {return advancedSearchStore.getValue()}`, you can write `props => advancedSearchStore.getValue()`

File: src/page/search/advanced-search/README.md [hunk 3]
```diff
@@ -168,6 +168,57 @@ Evénement à définir par l'utilisateur.
...
+	        {key: "ATTRIBUT0", order: true, label: "Trier par attribut 0")}
+	    ]
+	} else if (this.props.scope==="scope1") {
+	    orderableColumnList = [
+	        {key: "ATTRIBUT1", order: true, label: "Trier par attribut 1")}
+	    ]
+	}
+	
+	return (
+	    <div data-vision="recherche">
```
Comment (pierr) [inline src/page/search/advanced-search/README.md]: is this div usefull for the point you are trying to make ....  Not sure....

**Comment by TomGallon** [comment]
copied in https://github.com/KleeGroup/focus-components/pull/1286

**Pull request closed without merging**